ArmoniK.SDK.Worker.Test/include/SegFaultService.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <armonik/sdk/worker/ServiceBase.h>
+#include <iostream>
+namespace ArmoniK {
+namespace Sdk {
+namespace Worker {
+namespace Test {
+
+/**
+ * @brief Example implementation of a ArmoniK::Sdk::Worker::ServiceBase
+ */
+
+/**
+ * @brief Example implementation of a ArmoniK::Sdk::Worker::ServiceBase
+ */
+class SegFaultService : ServiceBase {
+public:
+  std::string call(void *, const std::string &name, const std::string &input) override {
+    std::cout << "SegFaultService method : " << name << std::endl;
+    double *compute = nullptr;
+    compute[0] = (double)std::rand() / RAND_MAX;
```

Comment:
The code could just assign a value (1 for example) instead of calling rand

Reply:
```cpp
*(volatile int*)0 = 0;
```